README.md

```diff
@@ -1,7 +1,37 @@
 # LSTM sentiment analysis on the Yelp-dataset
 
+This project does sentiment analysis on the [yelp-review](https://www.yelp.com/dataset) dataset. This is done by modeling the star rating and the number of votes for the review being _cool_, _funny_ and _helpful_ as a function of the written text in the review.
+
+The models evaluated are different combinations of recurrent neural networks, embeddings and convolutional neural networks. All the models are implemented in [PyTorch](http://pytorch.org/).
+
+
+## Visualization
+For visualization of training and other data [Visdom](https://github.com/facebookresearch/visdom) is used. This means that you have to start a Visdom server before being able to visualize anything.
```

Comment:
Good to point out. Perhaps we could add a flag to enable visualization for later versions.